zebra-chain/src/history_tree.rs

```diff
@@ -414,22 +414,28 @@ impl Clone for NonEmptyHistoryTree {
 
 /// A History Tree that keeps track of its own creation in the Heartwood
 /// activation block, being empty beforehand.
-#[derive(Debug, Default, Clone)]
+#[derive(Debug, Default)]
```

Comment:
Nitpick: Could we restore the derived `Clone` impl?

Reply:
restored